tests/numerics/type_tensor_test.C

```diff
@@ -23,7 +23,9 @@ public:
   CPPUNIT_TEST(testRotation);
 #endif
   CPPUNIT_TEST(testIsZero);
-
+#ifdef LIBMESH_HAVE_METAPHYSICL
+  CPPUNIT_TEST(testCompareTypes);
```

Comment:
If you want to wrap these in `ifdef LIBMESH_HAVE_METAPHYSICL` I would suggest renaming to `testReplaceAlgebraicType` since `CompareTypes` is always defined in libMesh.

Reply:
Renamed them!